tensorflow/compiler/xla/tests/hlo_test_base.h

```diff
@@ -274,6 +274,7 @@ class HloTestBase : public ::testing::Test {
   // inspect a particular computation or instruction.
   HloComputation* FindComputation(HloModule* module, absl::string_view name);
   HloInstruction* FindInstruction(HloModule* module, absl::string_view name);
+  HloInstruction* FindInstruction(HloModule* module, HloOpcode opcode);
```

Comment:
The comment before the group is for finding the instruction with the given name. Can you modify the comment to including finding the instruction with the given opcode?